api/src/main/java/org/eclipse/microprofile/lra/client/LRAInfo.java

```diff
@@ -36,28 +36,27 @@
      */
     String getClientId();
 
-    /**
-     * @return  true if lra was successfully completed, false otherwise
-     */
-    boolean isComplete();
-
-    /**
-     * @return  true if lra was compensated, false otherwise
-     */
-    boolean isCompensated();
-
     /**
      * @return  true if recovery is in progress on the lra, false otherwise
      */
     boolean isRecovering();
 
     /**
-     * @return  true if lra is in active state right now, false otherwise
+     * Test if the LRA has not been asked to close or cancel.
+     * @return  true if lra is in active state right now, false otherwise.
      */
     boolean isActive();
 
     /**
      * @return  true if lra is top level (not nested), false otherwise
      */
     boolean isTopLevel();
+
+    /**
+     * The current status of this LRA. Valid values match
+     * {@link org.eclipse.microprofile.lra.annotation.CompensatorStatus}
+     * @return the status of the LRA. A null value or the empty string
+     * means that the LRA is still active ({@link LRAInfo#isActive()})
+     */
+    String getStatus();
```

Comment:
+1

Reply:
I agree this is better than the different is* methods

Reply:
LRAInfo is mainly used in JAX-RS responses and String was chosen for simplicity/experdiency and it avoids any extra complexity for clients to have deal with the resulting JSON.